Plots/Masking/mask_1.py

```diff
@@ -0,0 +1,187 @@
+"""
+mask_1.py
+===============
+Demonstrates the use of the xarray's where function and 
+a masking array to mask out land or ocean.
+
+https://www.ncl.ucar.edu/Applications/Scripts/mask_1.ncl
+
+https://www.ncl.ucar.edu/Applications/Images/mask_1_1_lg.png
+
+
+https://www.ncl.ucar.edu/Applications/Images/mask_1_2_lg.png
+"""
+
+import cartopy.crs as ccrs
+import cmaps
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+
+# Read in the netCDF file
+
+ds = (
+    xr.open_dataset("../../data/netcdf_files/atmos.nc", decode_times=False)
+    .isel(time=0)
+    .drop("time")
+)
+
+
+# Use xarray's where function to mask out land and then ocean data
+land_only = ds.TS.where(ds.ORO == 1.0)
+ocean_only = ds.TS.where(ds.ORO == 0.0)
+
+
+# Define a few utility functions
+
+def truncate_colormap(cmap, minval=0.0, maxval=1.0, n=100):
```

Comment:
Could you please explain  a little more about what truncate_colormap is doing (and maybe why it is needed)

Reply:
Added a note in the documentation string of the function